dace/frontend/python/newast.py

```diff
@@ -28,6 +29,7 @@
 from dace.properties import LambdaProperty, CodeBlock
 from dace.sdfg import SDFG, SDFGState
 from dace.symbolic import pystr_to_symbolic
+from dace.sourcemap import temporaryInfo
```

Comment:
please avoid importing functions/classes directly (we try to avoid this for any new code that gets added). See coding guidelines.